src/components/Property/RaiseWorkOrder/RaiseWorkOrderFormView.js

```diff
@@ -133,34 +133,45 @@ const RaiseWorkOrderFormView = ({ propertyReference }) => {
     setError(null)
 
     try {
-      const data = await frontEndApiRequest({
-        method: 'get',
-        path: `/api/properties/${propertyReference}`,
-      })
-      const priorities = await frontEndApiRequest({
-        method: 'get',
-        path: `/api/schedule-of-rates/priorities`,
-      })
-      const trades = await frontEndApiRequest({
-        method: 'get',
-        path: `/api/schedule-of-rates/trades?propRef=${propertyReference}`,
-      })
-      const user = await frontEndApiRequest({
-        method: 'get',
-        path: '/api/hub-user',
-      })
+      const [propertyResponse, priorities, trades, user] = await Promise.all([
+        frontEndApiRequest({
+          method: 'get',
+          path: `/api/properties/${propertyReference}`,
+        }),
+        frontEndApiRequest({
+          method: 'get',
+          path: `/api/schedule-of-rates/priorities`,
+        }),
+        frontEndApiRequest({
+          method: 'get',
+          path: `/api/schedule-of-rates/trades?propRef=${propertyReference}`,
+        }),
+        frontEndApiRequest({
+          method: 'get',
+          path: '/api/hub-user',
+        }),
+      ])
+
+      let contactDetails = []
+
+      if (propertyResponse?.tenure?.id != null) {
+        contactDetails = await frontEndApiRequest({
+          method: 'get',
+          path: `/api/contact-details/${propertyResponse?.tenure?.id}`,
+        })
+      }
```

Comment:
Fetch contact details if tenureId is present. This is the same implementation used on the properties endpoint.